code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1175,15 +1175,39 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		to_chat(H, "<span class='danger'>You feel weak.</span>")
 
 	if(radiation > RAD_MOB_VOMIT && prob(RAD_MOB_VOMIT_PROB))
-		H.vomit(10, TRUE)
+		var/obj/item/organ/stomach = H.getorganslot( ORGAN_SLOT_STOMACH )
+		if( !( H.getorgan( /obj/item/organ/stomach ) && stomach.organ_flags == ORGAN_SYNTHETIC ) ) // NSV13 synthetics do not vomit
+			H.vomit(10, TRUE)
 
-	if(radiation > RAD_MOB_MUTATE)
+	if(!HAS_TRAIT( H, TRAIT_MUTATEIMMUNE ) && radiation > RAD_MOB_MUTATE)
 		if(prob(1))
 			to_chat(H, "<span class='danger'>You mutate!</span>")
 			H.easy_randmut(NEGATIVE+MINOR_NEGATIVE)
 			H.emote("gasp")
 			H.domutcheck()
 
+	if( HAS_TRAIT( H, TRAIT_IPCRADBRAINDAMAGE ) && radiation > RAD_MOB_MUTATE ) // NSV13 IPCs will gain brain damage instead of mutations when exposed to radiation
+		if ( prob( 1 ) )
```

Comment:
so its a 50% chance of firing each tick this fires for the mob? It would need a proper test merge to see how debilitating it could be.